web/demos/drupal-multisite/index.html

```diff
@@ -0,0 +1,164 @@
+  <link rel="apple-touch-icon" sizes="180x180" href="../../favicon/apple-icon-180x180.png">
+  <link rel="icon" type="image/png" sizes="192x192"  href="../../favicon/android-icon-192x192.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../../favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="../../favicon/favicon-96x96.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../../favicon/favicon-16x16.png">
+  <link rel="manifest" href="../../favicon/manifest.json">
+  <meta name="msapplication-TileImage" content="/favicon/ms-icon-144x144.png">
+
+  <style type="text/css">
+    /* Put presentation-specific CSS here. Yes, it will happen. */
+  </style>
+
+</head>
+
+<!-- ontouchstart="" will enable :hover and :active events on tap gesture for PWAs -->
+<body ontouchstart="">
+
+<div class="reveal">
+
+  <!-- Any section element inside of this container is displayed as a slide -->
+  <div class="slides">
+
+    <section id="title-slide" data-background-color="#4786ff">
+      <h1>Drupal 8 Multi-site on Platform.sh</h1>
+    </section>
+
+    <section>
+      <blockquote>What about Drupal multi-site?</blockquote>
+    </section>
+
+    <section>
+    <div class="two-col-svg">
+      <div>Setting up a Drupal 8 multi-site app with subdomains takes just a few modifications from our existing template.</div>
+      <div><img src="assets/druplicon.svg" class="plain" width="400px" alt="Composer" /></div>
+    </div>
+    </section>
+
+    <section>
+      <img src="assets/01-routes.png" alt="Multi-site routes." width="75%"/>
+      <p style="margin-top: 2em;">Each site gets a unique subsite ID</p>
+      <p>that is used to define a its subdomain in <code>routes.yaml,</code></p>
+    </section>
+
+    <section>
+      <img src="assets/02-settings.png" alt="Multi-site settings directories." />
+      <p style="margin-top: 2em;">their own <code>config/sync</code> directory as well as settings directories</p>
+      <p> in <code>sites</code>, which are identical to <code>default</code> to start,</p>
+      <p>but could be customized for each site.</p>
+    </section>
+
+    <section>
+      <img src="assets/03-relationships.png" alt="Subsite relationships." />
+      <p style="margin-top: 2em;">Each subsite has its own database</p>
+      <p>and endpoint, so they get their own relationships</p>
+      <p>in <code>.platform.app.yaml</code>.</p>
+    </section>
+
+    <section>
+      <img src="assets/04-services.png" alt="Database definitions." width="60%"/>
+      <p style="margin-top: 2em;">Each subsite's database is defined for the same</p>
+      <p>MariaDB instance in the <code>services.yaml</code> file.</p>
+    </section>
+
+    <section>
+      <video width="80%" data-autoplay muted playsinline>
+        <source src="assets/05-urls.mp4" type="video/mp4">
+      </video>
+      <p>To give you a separate Drupal 8 installation at each subdomain.</p>
```

Comment:
Again, the sentence fragment doesn't flow well. If you really want to have sentences split across slides, I would pick a style: either separate phrases or bullet-point style sentence fragments, not both.